lib/advanced-open-file-view.coffee

```diff
@@ -244,6 +245,7 @@ class AdvancedFileView extends View
       newPath = ''
 
     @miniEditor.setText(newPath)
+    @miniEditor.getModel().scrollToCursorPosition()
```

Comment:
Speaking of which, with that commit you can just call `@scrollToCursor()` here and in `undo` instead of calling it directly.